docs/built_in_extensions.rst

```diff
@@ -108,6 +108,37 @@ The HTTP entrypoint is built on top of `werkzeug <http://werkzeug.pocoo.org/>`_.
     Date: Fri, 13 Feb 2015 14:58:48 GMT
 
 
+Exceptions can be raised safely if added to ``expected_exceptions``. You can also change exceptions formatting by overriding :meth:`~nameko.web.HttpRequestHandler.response_from_exception`:
```

Comment:
"exception formatting" or possibly "exception's formatting". though maybe explicitly mention that this controls how exceptions are returned to a http caller

i'm not sure "safely" is appropriate here. do we write about `expected_exceptions` elsewhere?